### PR TITLE
Download the template from the examples repo

### LINKED ISSUE #22: Let the template generator download the template from the examples repo
Instead of having the template inside this repository, we should just download the zip from the examples repo.

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Fix #22

Also updated eslint to use 9.x, bumped minimum node version (for fetch)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,13 +107,7 @@ Once Restate is up, register the service deployment in Restate by executing:
...
 
 ```shell
-curl -X POST http://localhost:8080/myservice/hello -H 'content-type: application/json' -d '{"request": "Pete"}'
-```
-
-For the gRPC-based template, use the following command instead:
-
-```shell
-curl -X POST http://localhost:8080/org.example.ExampleService/SampleCall -H 'content-type: application/json' -d '{"request": "Pete"}'
+curl -X POST http://localhost:8080/myservice/hello -H 'content-type: application/json' -d '"Pete"'
```
Comment (gvdongen) [inline README.md:110]:
```suggestion
curl localhost:8080/myservice/hello -H 'content-type: application/json' -d '"Pete"'
```
```suggestion
curl localhost:8080/myservice/hello -H 'content-type: application/json' -d '"Pete"'
```

**Review by gvdongen — APPROVED** [review]
Looks good to me but we also need to update the comments in the template itself: https://github.com/restatedev/examples/blob/dev_0.9/templates/typescript/src/app.ts

Or make sure they don't have code in them because they are constantly outdated....

**Comment by gvdongen** [comment]
> Looks good to me but we also need to update the comments in the template itself: https://github.com/restatedev/examples/blob/dev_0.9/templates/typescript/src/app.ts
> 
> Or make sure they don't have code in them because they are constantly outdated....

It should just refer to the quickstart actually. Should I pick this up?

**Comment by slinkydeveloper** [comment]
> It should just refer to the quickstart actually. Should I pick this up?

That would be great. Perhaps the same should be done for java and ts

**Pull request merged**